components/brave_wallet/browser/solana_transaction.cc

```diff
@@ -262,11 +263,6 @@ SolanaTransaction::GetSignedTransactionBytes(
   }
   CompactU16Encode(signers.size(), &transaction_bytes);
 
-  const auto selected_account = keyring_service->GetSelectedSolanaDappAccount();
-  if (!selected_account) {
-    return std::nullopt;
-  }
-
```

Comment:
That actually was the only place where `GetSelectedSolanaDappAccount` was called outside of `SolanaProviderImpl`. So now it matches with `GetSelectedEthereumDappAccount` which is needed only for `EthereumProviderImpl`